NEWS: agent: suppress annoying "registration != duplicate" warning for root oids
(otherwise, you'll get this message on almost every startup)

http://net-snmp.svn.sourceforge.net/viewvc/net-snmp?view=rev&revision=16669

--- branches/V5-4-patches/net-snmp/agent/agent_registry.c	2007/08/20 08:06:42	16668
+++ branches/V5-4-patches/net-snmp/agent/agent_registry.c	2007/08/22 21:56:23	16669
@@ -532,7 +532,8 @@
 	
 	    if (next && (next->namelen  == new_sub->namelen) &&
 		(next->priority == new_sub->priority)) {
-                netsnmp_assert(!"registration != duplicate"); /* always false */
+                if (new_sub->namelen != 1) /* ignore root OID dups */
+                    netsnmp_assert(!"registration != duplicate"); /* always false */
 		return MIB_DUPLICATE_REGISTRATION;
 	    }
 
